acroread

```diff
@@ -13,13 +14,19 @@ main() {
         fi
     fi
 
+    if [[ $(type -p "ifconfig") ]]; then
```

Comment:
wow, really sorry I haven't seen this until now. might not be relevant any more...
i would suggest checking the following way:
`if type -p ifconfig &> /dev/null; then`